redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/YogaUIView.kt

```diff
@@ -45,7 +49,54 @@ internal class YogaUIView : UIScrollView(cValue { CGRectZero }), UIScrollViewDel
     contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentNever
   }
 
-  override fun intrinsicContentSize(): CValue<CGSize> = calculateLayout()
+  /**
+   * The intrinsic size is broken by design if any subview's height depends on its width (or
+   * vice-versa). For example, if a subview is UILabel that wraps, we need to know how wide the
+   * label is before we can compute that label's height.
+   *
+   * We work around this by:
+   *
+   *  1. Making [intrinsicContentSize] depend on the mostly-recently applied bounds
+   *  2. Invalidating it each time the bounds change
```

Comment:
This work around sounds good to me, though @edahle do you know how a `UITextView` and other views that depend on knowing the width to compute the height provide a reasonable `intrinsicContentSize`? Maybe private APIs of some kind?